distribution/openrct2.d.ts

```diff
@@ -449,6 +449,8 @@ declare global {
 
         addition: number | null;
         isAdditionBroken: boolean;
+
+        direction: 0 | 1 | 2 | 3;
```

Comment:
It might be nice to declare a type for this now. E.g.:
`type Direction = 0 | 1 | 2 | 3`

Reply:
Good call, definitely clearer.